<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterAttendance</class>
 <widget class="QWidget" name="RegisterAttendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/assets/assets/plan.jpg);
font-size: 16px;
color: black;
background-color: white;</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>651</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 25px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="emailInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;
font: italic;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="startTimeLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 25px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Start Time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="startTimeInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;
font: italic;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="closeTimeLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 25px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Close Time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="closeTimeInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;
font: italic;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="styleSheet">
       <string notr="true">color: purple;
font-size: 25px;
font: Arial;</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="dateInp">
      <property name="styleSheet">
       <string notr="true">height: 25px;
color: black;
font: italic;
font-size: 16px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 1;
 padding: 10px;
 box-shadow: 2px 2px grey;
 font-size: 18px;
 color: white;
 background: green; 
border-radius: 5px;
z-index: 1;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>29</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;h2 style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#0000ff;&quot;&gt; Register Staff Attendance&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 1;
 padding: 10px;
 box-shadow: 2px 2px grey;
 font-size: 18px;
 color: white;
 background: red; 
border-radius: 5px;
z-index: 1;
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
